macros/snowplow_media_player_base_events_this_run/media_ad_break_field.sql

```diff
@@ -12,6 +12,12 @@ You may obtain a copy of the Snowplow Personal and Academic License Version 1.0
         col_prefix='contexts_com_snowplowanalytics_snowplow_media_ad_break_1'
       ) }}
     {%- else -%}
-      null
+      {% if property is string and target.type not in ['postgres', 'redshift'] -%}
+          {{ property }}
```

Comment:
Just a comment for future refactor if we look back, "it’s so that in later calls of the macro (where we use the column name only, not provide a full type definition) we carry through the type of the null"